serverless/src/migrateDatabase/handler.js

```diff
@@ -11,6 +11,7 @@ let dbConnectionConfig = null
 const migrateDatabase = async (event) => {
   try {
     dbConnectionConfig = await getDbConnectionConfig(dbConnectionConfig)
+    dbConnectionConfig.database = 'uat-import-test'
```

Comment:
Is this the database you used locally? Does it need to be committed?

Reply:
Yea, this will likely just be deleted after the production migration but I wanted to make sure the current state of the script lived in source control -- these lambas won't be run in the application.